Examples/Examples/All Examples/SwiftUIExample.swift

```diff
@@ -0,0 +1,264 @@
+
+        let oldAnnotationIds = Set(mapView.annotationManager.annotations.values.map(\.identifier))
+        let newAnnotationIds = Set(annotationsByIdentifier.values.map(\.identifier))
+
+        let idsForAnnotationsToRemove = oldAnnotationIds.subtracting(newAnnotationIds)
+        let annotationsToRemove = idsForAnnotationsToRemove.compactMap { mapView.annotationManager.annotations[$0] }
+        if !annotationsToRemove.isEmpty {
+            mapView.annotationManager.removeAnnotations(annotationsToRemove)
+        }
+
+        let idsForAnnotationsToAdd = newAnnotationIds.subtracting(oldAnnotationIds)
+        let annotationsToAdd = idsForAnnotationsToAdd.compactMap { annotationsByIdentifier[$0] }
+        if !annotationsToAdd.isEmpty {
+            mapView.annotationManager.addAnnotations(annotationsToAdd)
+        }
+    }
+}
+
+/// Here's an example usasge of `SwiftUIMapView`
+internal struct ContentView: View {
+
+    /// For demonstration purposes, this view has its own state for the camera and style URL.
+    /// In your app, these values could be constants defined directly in `body` or could come
+    /// from a model object.
+    @State private var camera = Camera(center: CLLocationCoordinate2D(latitude: 40, longitude: -75), zoom: 14)
+    @State private var styleURL = StyleURL.streets
+
+    /// Each time you create an annotation, it is assigned a UUID. For this reason, it's not a great
+    /// idea to actually create annotations inside of `body` which may be called repeatedly
+    @State private var annotations = [
+        PointAnnotation(coordinate: CLLocationCoordinate2D(latitude: 40, longitude: -75)),
+        PointAnnotation(coordinate: CLLocationCoordinate2D(latitude: 40, longitude: -75.001)),
+        PointAnnotation(coordinate: CLLocationCoordinate2D(latitude: 40, longitude: -74.999))]
+
+    public var body: some View {
+        VStack {
+            SwiftUIMapView(
+                resourceOptions: ResourceOptions(accessToken: AccountManager.shared.accessToken!),
+
+                /// Here, we pass the camera state variable into `SwiftUIMapView` as a binding
+                camera: $camera)
+
+                /// Here's an example usage of the builder method to set `styleURL`.
+                /// Note that in this case, we just need the current value, so we write
+                /// `styleURL`, not `$styleURL`
+                .styleURL(styleURL)
+
+                /// Since these methods use the builder pattern, we can chain them together
+                .annotations(annotations)
+
+            /// We configure the slider to bind to the camera's zoom. Adjusting the slider with
+            /// change the zoom on the map, and changing the zoom by interacting with the map
+            /// will change the slider. Here's the data flow:
+            ///
+            /// Slider to Map:
+            ///     - User interacts with the slider
+            ///     - Slider updates the camera binding's zoom value
```

Comment:
Did something happen with indents here?

Reply:
This is how Xcode likes it. The comments are aligned to the indentation of the next line of code.